packages/web/docs/src/pages/self-hosting/get-started.mdx

```diff
@@ -89,23 +95,151 @@ the environment variables.
 
 </Callout>
 
-After setting the environment variables, you can start the GraphQL Hive services using
-`docker-compose`.
+After setting the environment variables, pull the required images for GraphQL Hive services. This's
+gonna take some time if you're doing it for the first time.
+
+```bash
+docker compose -f docker-compose.community.yml pull
+```
+
+After it's done, you can start the GraphQL Hive services using `docker compose`.
 
 ```bash
-docker-compose -f docker-compose.community.yml up
+docker compose -f docker-compose.community.yml up
 # or of you embedded the environment variables
-docker-compose -f docker-compose.with-env.yml up
+docker compose -f docker-compose.with-env.yml up
```

Comment:
I don't see `docker-compose.with-env.yml` mentioned above?

Reply:
oh so this's a modified version for people who included their env vars in the docker compose but I should indeed clarify that above the line.